bulk_reassign_docs.py

```diff
@@ -0,0 +1,173 @@
+    # HTTP and other errors are handled generally by raising them as exceptions to be surfaced in downstream code
+    except requests.exceptions.HTTPError as errh:
+        # The API response's message value is sent in lieu of the full response to display to the user for clarity
+        raise Exception(errh.response.status_code,
+                        f"API message: {errh.response.json()['message']}")
+    except requests.exceptions.ConnectionError as errc:
+        raise Exception(
+            f"Connection Error: {errc}, API response: {errc.response.text}")
+    except requests.exceptions.Timeout as errt:
+        raise Exception(
+            f"Timeout Error: {errt}, API response: {errt.response.text}")
+    except requests.exceptions.RequestException as err:
+        raise Exception(
+            f"Other Error: {err}, API response: {err.response.text}")
+    else:
+        data = response.json()
+        if len(data['entries']) == 0:
+            print("No users found with this email:", user_email)
+            raise SystemExit("Script aborted")
+        else:
+            return data['entries'][0]['memberId']
+
+
+def get_member_files(client, user_id):
+    """ Update member
+        :user_id:       id of current owner
+
+        :returns:       an array of file objects
+
+    """
+    data = []
+    moreResults = True
+    nextPage = ''
+    while moreResults:
+        try:
+            response = client.get(
+                f"v2/members/{user_id}/files?typeFilters=workbook&typeFilters=dataset&limit=500{nextPage}"
+            )
+            response.raise_for_status()
+
+        except requests.exceptions.HTTPError as errh:
+            raise Exception(
+                f"Connection Error: {errh}, API response: {errh.response.text}")
+        except requests.exceptions.ConnectionError as errc:
+            raise Exception(
+                f"Connection Error: {errc}, API response: {errc.response.text}")
+        except requests.exceptions.Timeout as errt:
+            raise Exception(
+                f"Timeout Error: {errt}, API response: {errt.response.text}")
+        except requests.exceptions.RequestException as err:
+            raise Exception(
+                f"Other Error: {err}, API response: {err.response.text}")
+        else:
+            resp = response.json()
+            data = data + resp['entries']
+            if resp['nextPage'] is None:
+                moreResults = False
+            else:
+                pageID = str(resp['nextPage'])
+                nextPage = f'&page={pageID}'
+    return data
+
+
+def update_file(client, user_id, file_id):
+    """ Update file
+
+        :access_token:  Generated access token
+        :userId:        ID of the new owner
+        :file_id:       File to transfer ownership of
+
+        :returns:       Response JSON
+
+    """
+    try:
+        response = client.patch(
+            f"v2/files/{file_id}",
+            json={"ownerId": user_id}
```

Comment:
Without a `parentId`, what happens with workbooks and unsaved explorations? Would these all be dumped into someone's "My documents" section? (I assume if the workbooks are in a workspace, then they won't be moved)

Could it be better to allow the end user to optionally designate a folder into which these reassigned files will go?